src/components/consumptionItem/index.tsx

```diff
@@ -6,28 +6,28 @@ interface Props {
 }
 
 const ConsumptionItem: React.FC<Props> = ({ consumption }) => {
-  let aantalZessen = 0;
-  let rest = 0;
-  function prepDice(prijs?: number | null) {
-    let tmpArray = [];
-    if (prijs != null) {
-      aantalZessen = ~~(prijs / 6);
-      rest = prijs % 6;
+  const renderDice = (ticketAmount?: number | null) => {
```

Comment:
@matthias-D Hier heb ik de functie wat anders geschreven (gebruik gemaakt van arrow notatie). Daarnaast heb ik een aantal variabelen verwijderd die niet nodig waren of beter op een andere plek gezet werden.

Reply:
@Admiral-Sanders  volledig mee akkoord,
Die `    if (prijs != null) {`had ik daar staan omdat ik anders een error kreeg over dat die 'prijs' al dan niet 0 kon zijn, ik kon da op geen andere manier oplossen